layers:
  - name: occupancy
    frame_id: map
    class: OccupancyGridLayer
    load_file: false
    subscribe_mode: true
    enable_update: false
